Astral Pathfinder/Game/UIManager.cpp

```diff
@@ -73,12 +69,104 @@ void UIManager::render(Game::State *gameState) {
     return;
   }
   
-  if (gameState->planetSelected) planetInfo.render();
+  shipInfo.render();
+  if(gameState->planetSelected) selectedPlanetInfo.render(pm->getSelectedPlanet().getPopulation());
+  if(gameState->planetCollided) currentPlanetInfo.render(pm->getCurrentPlanet().getPopulation());
 }
 
 // MARK: - UIManager Methods
 
 void UIManager::setSelectedPlanet(Planet p) {
-  planetInfo.setUiTextures(p);
+  selectedPlanetInfo.setUiTextures(p);
+}
+
+void UIManager::setCurrentPlanet(Planet p) {
+  currentPlanetInfo.setUiTextures(p);
+}
+
+void UIManager::handleMouseDown(Game::State *gs, PlanetManager *pm) {
+  // If mouse button not pressed down, don't check for slider movement
+  if(!gs->mouseDown) return;
+  
+  if(gs->mouseDown)
+    window = checkClickedArea(gs->clickLocation);
```

Comment:
I think the code could be changed to enhance readability in this section. It isn't clear what one and two means in regards to the sliders.